### PR TITLE
LHC-1407

### PR DESCRIPTION
**Pull request by rdai10**
### Description
Create a new template to display a warning for on specific pages if there is a DXP cloud outage happening. 

### Checklist
- [x] Code compiles without errors.
- [ ] Include tests for new features and functionality.
- [x] Update README/documentation, when applicable.
- [ ] All tests, new and existing, pass without errors.
- [x] Checked browser compatibility, especially IE11.

### CODE REVIEW

File: theme/documentation/article_pages.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ This article is to be used to promote knowledge sharing and to provide solutions
 
 This article is built on top of the **Knowledge Base Articles** designed to be part of Liferay's fast-track publication program of providing a repository of verified solutions that have resolved one or more customer issues. It contains a fast track label at the top of the article, as well as a brief explanation about what fast track articles are at the bottom of the article. Articles published under this template are part of the Knowledge-Centered Service initiative.
 
+### System Status article
+
+This is a one-off article template based on the Zendesk default article to display a system outage warning at the top of the page for DXP cloud.
```
Comment (jonmak08) [inline theme/documentation/article_pages.md]: We can update this to: `This is a one-off article template used to display a system outage warning at the top of the page for DXP cloud.  This is based on Zendesk's default article.`

File: theme/src/resources/templates/article_pages/system_status.hbs [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+	<div class="row">
+		<div class="col-sm-12" id="systemStatusBanner"></div>
+
+		<div class="col-sm-12">
+			{{breadcrumbs}}
+		</div>
+	</div>
+
+	{{#with article}}
+	<div class="row">
```
Comment (jonmak08) [inline theme/src/resources/templates/article_pages/system_status.hbs]: Each element should be indented 1 tab when it is nested within a handlebars template tag.  See https://github.com/rdai10/zendesk/blob/master/theme/src/resources/templates/article_page.hbs#L8...L80.

File: theme/src/resources/templates/article_pages/system_status.hbs [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+		<div class="article-sidenav col-md-3 secondary-font">
+			<div class="sidenav">
+				<div class="collapsible-sidebar">
+					<svg aria-label="Documentation Navigation Menu" class="lexicon-icon lexicon-icon-menu-hamburger"
```
Comment (jonmak08) [inline theme/src/resources/templates/article_pages/system_status.hbs]: Does the `aria-label` need to be translated?

File: theme/documentation/article_pages.md (hunk 1 above)
Comment (rdai10) [reply theme/documentation/article_pages.md]: ccb4116

File: theme/src/resources/templates/article_pages/system_status.hbs (hunk 2 above)
Comment (rdai10) [reply theme/src/resources/templates/article_pages/system_status.hbs]: 8e352c9

**Pull request merged**